src/index.ts

```diff
@@ -16,6 +16,23 @@ import { cleanupHandlers, writeHandlers } from "./wrapper";
 import { hasWebpackPlugin } from "./util";
 import { TracingMode } from "./templates/common";
 import { addCloudWatchForwarderSubscriptions } from "./forwarder";
+import { Package, Event } from "serverless";
+
+// Separate interface since DefinitelyTyped dependency currently doesn't include tags
+export interface FunctionDefinitionWithTags {
+  name: string;
+  package: Package;
+  runtime?: string;
+  handler: string;
+  timeout?: number;
+  memorySize?: number;
+  environment?: { [name: string]: string };
+  events: Event[];
+  tags?: { [key: string]: string };
+}
```

Comment:
I think it would probably be preferable to build the fixed interface off the incomplete DefinitelyTyped interface with something like
```
interface FunctionDefinitionWithTags extends FunctionDefinition {
  tags: {[key: string]: string}
}
```
just to keep as much as possible in line with the official types